artemis-distribution/src/main/assembly/dep.xml

```diff
@@ -72,9 +72,18 @@
          <includes>
             <include>org.apache.activemq:artemis-console:war</include>
          </includes>
-         <outputDirectory>web</outputDirectory>
-         <unpack>false</unpack>
-         <outputFileNameMapping>console.war</outputFileNameMapping>
+         <outputDirectory>web/console.war</outputDirectory>
+         <unpack>true</unpack>
+         <!-- Exclude these jar files from console since they are already provided in the broker distribution.
+              If any of these jars is dropped from the broker distribution they should be added back to the console. -->
+         <unpackOptions>
+            <excludes>
+               <exclude>**/guava*.jar</exclude>
+               <exclude>**/commons-io*.jar</exclude>
+               <exclude>**/commons-logging*.jar</exclude>
+               <exclude>**/failureaccess*.jar</exclude>
+            </excludes>
+         </unpackOptions>
```

Comment:
So if I understand, this effectively removes the console.war file from the assembly, and extracts its [trimmed] contents instead, i.e puts a directory called console.war in its place? (EDIT: Or I could have read further before commenting, and saw the doc explaining that is what it does, hehe :D)

Do we need to do anything with the config for that changing? Its geared around war files from memory?

@brusdev any thoughts around the switch?

Reply:
To be clear, there are some minor differences under the covers, but unexploded and exploded war files are functionally equivalent in Jetty.

Reply:
I'm testing it locally to check what is the impact for users that want to replace console.war with the hawtio.war.

Reply:
It LGTM and the exploded war provides an easy way to customize files like web.xml